### PR TITLE
make the verification step checked/ignored namespace configurable

### PR DESCRIPTION
**Pull request by bmeng**

### CODE REVIEW

File: deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml [hunk 1]
```diff
@@ -31,3 +31,10 @@ data:
       - MetricsClientSendFailingSRE
       - UpgradeNodeScalingFailedSRE
       - UpgradeClusterCheckFailedSRE
+    verification:
+      ignoredNamespaces:
+      - openshift-logging
+      namespacePrefixesToCheck:
+      - openshift
+      - kube
+      - default
```
Comment (dofinn) [inline deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml:40]: have we reviewed any redhat-* requirements?

Comment (bmeng) [reply deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml:40]: not yet, just copied the values from here. https://github.com/openshift/managed-upgrade-operator/blob/master/pkg/osd_cluster_upgrader/upgrader.go#L363 
Haven't seen any project with name redhat, but maybe can create spike for that.

Comment (dofinn) [reply deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml:40]: +1 we need to ascertain if any layered product alerts should block an osd upgrade.

Comment (bmeng) [reply deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml:40]: https://issues.redhat.com/browse/OSD-5365

**Pull request merged**